cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/AppFabricServiceMain.java

```diff
@@ -164,8 +164,8 @@ protected void addServices(Injector injector, List<? super Service> services,
       services.add(injector.getInstance(SystemWorkerServiceLauncher.class));
     }
 
-    // Optionally adds the master environment task
-    masterEnv.getTask().ifPresent(task -> services.add(new MasterTaskExecutorService(task, masterEnvContext)));
+    // Adds the master environment tasks
+    masterEnv.getTasks().forEach(task -> services.add(new MasterTaskExecutorService(task, masterEnvContext)));
```

Comment:
The MasterTaskExecutorService has a hardcode thread name. Since now we can run more than one master env task, it's better to have the thread name contains the task name as well. The `MasterEnvironmentTask` can have a getName method that default to the simple class name (`getClass().getSimpleName()`)